float64 time
float64[7] tau_J
float64[7] tau_J_d
float64[7] coriolis
float64[7] gravity
